module-db/tests/Helpers.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "Helpers.hpp"
+
+#include <Utils.hpp>
+#include <set>
+#include <algorithm>
+#include <utility>
+
+namespace
+{
+    std::string readContent(const std::string &filename) noexcept
+    {
+        auto getFileSize = [](FILE *fd) -> std::size_t {
+            std::fseek(fd, 0, SEEK_END);
+            const auto size = std::ftell(fd);
+            std::rewind(fd);
+            return size;
+        };
+
+        std::vector<char> fileContent;
+        if (const auto fp = std::fopen(filename.c_str(), "r")) {
+            const auto fileSize = getFileSize(fp);
+
+            fileContent.reserve(fileSize + 1);
+            std::fread(fileContent.data(), 1, fileSize, fp);
+            std::fclose(fp);
+            fileContent[fileSize] = '\0';
+            return fileContent.data();
+        }
+        return {};
+    }
+
+    std::vector<std::string> readCommands(const std::filesystem::path &filePath)
+    {
+        const auto fileContent = readContent(filePath.c_str());
+        std::string currentStatement{};
+        std::vector<std::string> statements{};
+
+        std::string line{};
+        for (const auto &c : fileContent) {
+            if (c != '\n') {
+                line += c;
+            }
+            else {
```

Comment:
A minor thing for the future: consider an early `break` instead of an `else`.

Reply:
Yeah, in this case it might be applicable. I used already implemented solution, and tried to not modify it to save some time.